src/styles/index.scss

```diff
@@ -1,3 +1,33 @@
+@import 'blocks/variables';
+@import 'blocks/header/header';
+@import 'blocks/header/nav';
+@import 'blocks/catalog/catalog';
+@import 'blocks/catalog/product-card';
+@import 'blocks/catalog/stars';
+
+* {
```

Comment:
It's better to move general styles to a separate file to keep the main file clean and clear